gamemodes/terrortown/gamemode/server/sv_main.lua

```diff
@@ -522,6 +505,7 @@ function GM:InitPostEntity()
 	LoadShopsEquipment()
 
 	MsgN("[TTT2][INFO] Shops initialized...")
+	isShopFallbackInitialized = true
```

Comment:
Can we rename those to `TTT2ShopFallbackInitialized` since they are global?

Reply:
Good idea.